bin/git-undo

```diff
@@ -1,6 +1,28 @@
 #!/usr/bin/env bash
 
 back="^"
```

Comment:
The `$back` is unused now. Please remove it too.

Reply:
Yep, should have caught that. Thanks.